src/main/java/uk/gov/hmcts/reform/professionalapi/controller/request/ProfessionalUserReqValidator.java

```diff
@@ -29,16 +29,14 @@ public static boolean isValidEmail(String email) {
         return false;
     }
 
-    public void validateRequest(String orgId, String showDeleted, String email, String status) {
-        if (null == orgId  && null == email && null == showDeleted) {
+    public void validateRequest(String orgId, String showDeleted, String status) {
+        if (null == orgId  && null == showDeleted) {
             throw new InvalidRequest("No input values given for the request");
         }
 
         if (!StringUtils.isEmpty(status)) {
             validateUserStatus(status);
         }
-
-        isValidEmail(email);
```

Comment:
Not saying this is wrong; however I would like to know: why are we NOT validating email?

Reply:
The method 'isValidEmail' is being called in the controller as 'email' was removed in the parameter of the above method call.